source/docs/software/vision-processing/wpilibpi/installing-the-image-to-your-microsd-card.rst

```diff
@@ -7,7 +7,7 @@ Getting the FRC Raspberry PI image
 ----------------------------------
 
 The image is stored on the GitHub release page for the FRC\ |reg| PI-gen repository:
-https://github.com/wpilibsuite/FRCVision-pi-gen/releases.
+https://github.com/wpilibsuite/WPILibPi/releases.
```

Comment:
I think the image below should be updated as well

Reply:
updated DNS